RELEASE.md

```diff
@@ -1,45 +1,16 @@
 # Release Process
 
-TODO: the release process is too manual and needs to be automated
-
 The release process consists of these steps:
-  1. Edit build.gradle and remove the `-SNAPSHOT` suffix from the `version` parameter
-  1. Add an entry to CHANGELOG.pm with description of changes in the new release
-  1. Commit as "Preparing release"
-  1. Run `make release`
-     * Make sure you have the right version 0.9.2 of Thrift installed: `thrift -version`
-  1. Once the artifacts are uploaded to Sonatype staging, release them to Maven Central
+  1. Add an entry to [`CHANGELOG.md`](CHANGELOG.rst) with description of changes in the new release
+  1. Commit your `CHANGELOG` changes
+  1. Tag as a version (`git tag 0.20.0`). Use the one from [`build.gradle`](build.gradle) as reference.
```

Comment:
in [zipkin release scripts](https://github.com/openzipkin/zipkin/blob/master/RELEASE.md) the manual tag has a format `release-M.N.P`, and the script creates another tag (I think). If here the manual tag is the only one, it should have the `v` prefix, a common convention for semver.

And as @pavolloffay said, I think the build should match on the vM.N.P pattern when publishing a release, instead of any tag.